app/config/config.yml

```diff
@@ -0,0 +1,59 @@
+imports:
+    - { resource: parameters.yml }
+    - { resource: security.yml }
+    - { resource: services.yml }
+
+# Put parameters here that don't need to change on each machine where the app is deployed
+# http://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
```

Comment:
Not sure these fit this definition, but how crate/core config will be done is something we can iterate on.